.github/workflows/codecov.yaml

```diff
@@ -12,7 +12,7 @@ jobs:
     runs-on: ubuntu-latest
     timeout-minutes: 15
     steps:
-      - uses: actions/checkout@44c2b7a8a4ea60a981eaca3cf939b5f4305c123b # v3.5.2
+      - uses: actions/checkout@a5ac7e51b41094c92402da3b24376905380afc29 # v3.5.2
```

Comment:
These action version comments aren't getting updated. Either Dependabot needs to be able to manage them or we should remove them.